litmus-portal/self-deployer-manifest.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
```

Comment:
Why this manifest is redundant?

Reply:
This manifest is not redundant but, I have kept it separate from the k8s-manifest as it will clutter the k8s-manifest.